api/namex/models/request.py

```diff
@@ -633,7 +633,7 @@ def on_insert_or_update_nr(mapper, connection, request):
        
        Temporary NRs (nrNum starting with 'NR L') are discarded.
     """
-    if not request.nrNum.startswith('NR L'):
+    if current_app.config.get('DISABLE_NAMEREQUEST_NATS_UPDATES', 0) != 1 and not request.nrNum.startswith('NR L'):
```

Comment:
Is it OK that the fallback value for the "get" is 0? That would cause this condition to pass if you can't get the config, which doesn't seem safe...?

Reply:
If it is 0, then it will be different of 1 and it will be true. I just want to disable it when someone intentionally add it to the environment vars. That will be helpful for testing locally without a NATS server. The same thing is done for SOLR and NRO updates.

Reply:
This conversation can be closed. (I do not have permission to do so.)